test/language/statements/using/syntax/using-invalid-for-using-of-of.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2011 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: >
+    using: 'for (using of' is always interpreted as identifier 
+negative:
+  phase: parse
+  type: SyntaxError
+features: [explicit-resource-management]
+---*/
+
+$DONOTEVALUATE();
+for (using of of [1, 2, 3]) { }
```

Comment:
Could you add a new test case when `for (using of` starts a for-variable-declaration, e.g.
```js
for(using of = null;;;);
```

This should be valid according to current spec, but a few implementations reject it as if it always start a for-of statement.

c.f. https://github.com/tc39/proposal-explicit-resource-management/issues/248

Reply:
Fixed in [fc3079a](https://github.com/tc39/test262/pull/4483/commits/fc3079a585d827fdd094f323674e1e2d66f531d4)